importsrv/testtools_test.go

```diff
@@ -0,0 +1,102 @@
+package importsrv_test
+
+import (
+	"github.com/axiomhq/hyperloglog"
+	"github.com/stripe/veneur/samplers/metricpb"
+	"github.com/stripe/veneur/tdigest"
+)
+
+//
+// PROTOBUF
+//
+
+func pbmetrics(ms ...*metricpb.Metric) []*metricpb.Metric {
```

Comment:
heh, that's a neat trick for avoiding the `[]*metricpb.Metric{ ... }` syntax grossness.